File: ChatClient/ChatClient/model/dto/MessageDTO.cs
namespace ChatClient.model.dto
{
    internal class MessageDTO
    {
        private string _MessageText;

        public string MessageText
        {
            get
            {
                return _MessageText;
            }
            set
            {
                if (value != null)
                {
                    _MessageText = value;
                }
                else
                {
                    throw new InvalidInputException("messageText field is mandatory");
                }
            }
        }

        public MessageDTO(string messageText)
        {
            MessageText = messageText;
        }
    }
}

File: authdemo/src/main/java/com/example/authdemo/service/dto/MessageDto.java
package com.example.authdemo.service.dto;

import lombok.Data;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.util.Date;

@Data
public class MessageDto {

    private Long id;

    private String senderUsername;

    private String receiverUsername;

    @NotEmpty(message = "messageText is mandatory")
    private String messageText;

    private Date creationTime;

    private Date modificationTime;
}
